### ISSUE TITLE
ng new git error -- Too many arguments

### ISSUE THREAD

**Issue by cjones26** [ngCli/ng-cli#1]
Hey there--first of all thanks for the great work here!  I'm running git version 1.9.4.msysgit.2 on Windows with the latest version of ng-cli.  When attempting to create a project like so:

`ng new projectName`

I receive the following error after selecting a project blueprint (Default template or UI Router template):

``Too many arguments.``

From what I've seen this may be an issue with an extra space being generated as I am prompted with git usage after the error is displayed.  Any thoughts would be greatly appreciated!

**Comment by cjones26** [comment]
I've determined the cause of the issue--I was attempting to clone into a directory with spaces in it--therefore my path like:

C:\VS Projects\Angular\projectName would not work as git believes it has received too many arguments due to the path not being surrounded with quotes.

**Comment by thetutlage** [comment]
What happens when you run the following command 

```
git clone https://github.com/ngCli/ngcli-default-blueprint.git C:\VS Projects\Angular\projectName
```

**Comment by thetutlage** [comment]
Fixed as of v0.0.7
